.pre-commit-config.yaml

```diff
@@ -92,6 +92,16 @@ repos:
         entry: python3 -m tools.mypy_helpers.check_stronglist
         files: ^pyproject\.toml$
         language: python
+      - id: eslint
+        name: eslint
+        entry: ./node_modules/.bin/eslint
+        language: node
```

Comment:
for now, these should be `language: system` or `language: script` -- otherwise this'll provision a throwaway js environment for nothing -- ideally these would be managed by pre-commit itself but that's more work than just moving them over

Reply:
https://github.com/pre-commit/mirrors-eslint

Any reason we shouldn't use this? Or does it make more sense to just use the one yarn brings in?

Reply:
```
language: system
```

will use the volta node right?

Reply:
yeah it'll use the one in the current user's environment -- I think for now (despite a manage environment typically being better) this is the most apples-to-apples conversion of the current script

Reply:
pre-commit mirror didn't have the version we have specified in package.json, and also i wasn't sure if there was a pre-commit-supported way to use the version in package.json (is there?)